zenoh/src/query.rs

```diff
@@ -280,6 +280,7 @@ impl<'a, 'b, Handler> GetBuilder<'a, 'b, Handler> {
 
     /// Restrict the matching queryables that will receive the query
     /// to the ones that have the given [`Locality`](crate::prelude::Locality).
+    #[cfg(feature = "unstable")]
```

Comment:
`Locality` should be only on subscriber and queryable side for the time being at least.